### PR TITLE
[mobile] Fix dialog to GUI feedback

### PR DESCRIPTION
**Pull request by DohyungLim**
- No title Dialog : height value is fixed.
- 3 Button title Dialogs : ditto.
- Single choice description list Dialogs : 8 padding fixed.
- Multi choice Dialogs : left, right padding and text size fixed.

### CODE REVIEW

File: src/css/profile/mobile/common/popup.less [hunk 1]
```diff
@@ -187,8 +186,8 @@
 			content: "";
 			position: absolute;
 			width: @px_base;
-			height: 22 * @px_base;
-			top: 7 * @px_base;
+			height: 16 * @px_base;
+			top: 10 * @px_base;
 			background-color: var(--on-off-switch-divider-color);
 			opacity: var(--on-off-switch-divider-opacity);
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: Need to change the opacity to use the number between 0 and 1.

File: src/css/profile/mobile/common/popup.less [hunk 2]
```diff
@@ -124,12 +124,11 @@
 
 		ul .ui-li-has-checkbox {
 			& .ui-li-text span {
-				font-size: 16 * @px_base;
-				line-height: 23 * @px_base;
+				font-size: 18 * @px_base;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: Use @sp_base for the font-size.

File: src/css/profile/mobile/common/popup.less [hunk 3]
```diff
@@ -124,12 +124,11 @@
 
 		ul .ui-li-has-checkbox {
 			& .ui-li-text span {
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: Don't need the '&'.

File: src/css/profile/mobile/common/popup.less [hunk 4]
```diff
@@ -124,12 +124,11 @@
...
-				font-size: 16 * @px_base;
-				line-height: 23 * @px_base;
+				font-size: 18 * @px_base;
 			}
 			& input {
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: ditto.

File: src/css/profile/mobile/common/popup.less (hunk 1 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: I will change GUI guide value.

done.

File: src/css/profile/mobile/common/popup.less (hunk 2 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: done.

File: src/css/profile/mobile/common/popup.less (hunk 3 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: done.

File: src/css/profile/mobile/common/popup.less (hunk 4 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: done.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**